src/python/deepgnn/graph_engine/snark/tests/e2e_test.py

```diff
@@ -269,24 +269,30 @@ def json_to_edge_list_helper(json_input):
         else:
             return f"{json_input['src_id']},{json_input['dst_id']},{json_input['edge_type']},{json_input['weight']},{features}"
 
-    partition_meta = ""
     for i, p in enumerate(partitions):
         decoder = EdgeListDecoder()
         writer = BinaryWriter(output_dir, i)
         for v in p:
             writer.add(decoder.decode(json_to_edge_list_helper(v)))
         writer.close()
-        nf = "\n".join(map(str, writer.node_type_count))
-        ef = "\n".join(map(str, writer.edge_type_count))
-        if nf == "":
-            nf = "0\n0\n0"
-        if ef == "":
-            ef = "0\n0"
-        partition_meta += f"{i}\n3\n3\n3\n2\n2\n{nf}\n{ef}\n"
-    meta = open(os.path.join(output_dir, "meta.txt"), "w+")
-    meta.write(f"v3\n3\n3\n2\n15\n15\n2\n")
-    meta.write(partition_meta)
-    meta.close()
+    content = {
+        "binary_data_version": "v2",  # converter version
+        "node_count": 3,
+        "edge_count": 3,
+        "node_type_num": 3,
+        "edge_type_num": 2,
+        "node_feature_num": 15,
+        "edge_feature_num": 15,
+        "n_partitions": 2,  # partition count
```

Comment:
Please, remove n_partitions everywhere..

Reply:
You are somehow reviewing old code